master/usage/ip-pool-manipulation.md

```diff
@@ -0,0 +1,54 @@
+---
+title: IP Pool Manipulation
+---
+This document describes how to make common modifications to Calico's
```

Comment:
Actually reading the rest of the doc, I'm not really convinced I see the point of this doc.  What's the aim of the doc?
-  It reads more like a general document describing how to reconfigure a resource rather than being IP Pool specific.
-  If it is supposed to be IP Pool specific, then we really need to describe more about pools themselves.
-  Should this include creating new IP pool resources rather than just modifying them?